opal/mca/pmix/external/Makefile.am

```diff
@@ -19,8 +22,10 @@ noinst_LTLIBRARIES = libmca_pmix_external.la
 libmca_pmix_external_la_SOURCES = \
         external.h \
         pmix_external_component.c
+
+libmca_pmix_external_la_LIBADD = $(pmix_external_WRAPPER_EXTRA_LIBS)
 libmca_pmix_external_la_LDFLAGS = \
-    -module -avoid-version
+    -module -avoid-version $(pmix_external_WRAPPER_EXTRA_LDFLAGS)
```

Comment:
The wrapper flags are for the wrappers.  You probably shouldn't be using them in internal Makefile.am's.